target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -127,6 +130,10 @@ case "$FIRMWARE" in
 		caldata_extract "ART" 0x1000 0x2f20
 		ath10k_patch_mac $(macaddr_add "$(cat /sys/class/net/eth0/address)" 2)
 		;;
+	yinuolink,yn-q418)
+	        caldata_extract "ART" 0x1000 0x2f20
+                ath10k_patch_mac $(macaddr_add "$(hexdump -v -n 6 -s 0x1006 -e '5/1 "%02x:" 1/1 "%02x"' /dev/mtd7)" 0)
+                ;;
```

Comment:
Indent broken.
What are you trying to do with the MAC address here?
Why don't you just take the address for caldata?

Reply:
Why did you mark all these unresolved changes as resolved?

Reply:
sorry,wrong click